applications/CompressiblePotentialFlowApplication/custom_processes/move_model_part_process.cpp

```diff
@@ -45,31 +48,45 @@ MoveModelPartProcess::MoveModelPartProcess(ModelPart& rModelPart,
         mRotationPoint = mOrigin;
     }
     mRotationAngle = ThisParameters["rotation_angle"].GetDouble();
+    mRotationAxis = ThisParameters["rotation_axis"].GetVector();
     mSizingMultiplier = ThisParameters["sizing_multiplier"].GetDouble();
 }
 
 void MoveModelPartProcess::Execute()
 {
     KRATOS_TRY;
 
-    #pragma omp parallel for
-    for(int i = 0; i <  static_cast<int>(mrModelPart.NumberOfNodes()); ++i) {
+    Matrix translation_matrix = ZeroMatrix(4,4);
+    GeometricalTransformationUtilities::CalculateTranslationMatrix(1.0, translation_matrix, mOrigin);
+
+    Matrix rotation_matrix = ZeroMatrix(4,4);
+    GeometricalTransformationUtilities::CalculateRotationMatrix(mRotationAngle, rotation_matrix, mRotationAxis, mRotationPoint);
+
+    IndexPartition<size_t>(mrModelPart.NumberOfNodes()).for_each(
```

Comment:
I think you can use the `block_for_each` here, the index is a leftover from OpenMP